stable/prometheus-operator/Chart.yaml

```diff
@@ -10,7 +10,7 @@ name: prometheus-operator
 sources:
   - https://github.com/coreos/prometheus-operator
   - https://coreos.com/operators/prometheus
-version: 6.16.0
+version: 6.17.0
```

Comment:
Since this is a bugfix, only the patch version should be changed

Reply:
Thank you. Sorry for mistake.